src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -105,7 +105,9 @@ public async Task<IActionResult> Applications()
 
             var signinId = await _userService.GetSignInId();
             var applications = await _apiClient.GetApplications(signinId, false);
-            applications = applications.Where(app => app.ApplicationStatus != ApplicationStatus.Rejected && app.ApplicationStatus != ApplicationStatus.Cancelled).ToList();
+
+            var statusFilter = new[] { ApplicationStatus.Rejected, ApplicationStatus.Cancelled, ApplicationStatus.Withdrawn };
```

Comment:
leave for now, but if we're reusing this, or it's possibly going to be more complex, we need a static service that returns this list

Reply:
agreed. This is going to get expanded on soon